src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -3034,21 +3034,21 @@ namespace SurfaceGeometry {
         // Check if base surface and subsurface have the same normal
         Vectors::CompareTwoVectors(baseSurface.NewellSurfaceNormalVector, subSurface.NewellSurfaceNormalVector, sameSurfNormal, 0.001);
         if (sameSurfNormal) { // copy lcs vectors
-            // Prior logic tested for azimuth difference < 30 and then skipped this - this caused large diffs in
-            // CmplxGlz_MeasuredDeflectionAndShading Restoring that check here but will require further investigation (MJW Dec 2015)
-            if (std::abs(baseSurface.Azimuth - subSurface.Azimuth) > warningTolerance) {
-                subSurface.lcsx = baseSurface.lcsx;
-                subSurface.lcsy = baseSurface.lcsy;
-                subSurface.lcsy = baseSurface.lcsy;
-            }
+                              // Prior logic tested for azimuth difference < 30 and then skipped this - this caused large diffs in
+                              // CmplxGlz_MeasuredDeflectionAndShading Restoring that check here but will require further investigation (MJW Dec 2015)
+                              // if (std::abs(baseSurface.Azimuth - subSurface.Azimuth) > warningTolerance) {
+            subSurface.lcsx = baseSurface.lcsx;
+            subSurface.lcsy = baseSurface.lcsy;
+            subSurface.lcsz = baseSurface.lcsz;
+            // }
         } else {
-            // Not sure what this does, but keeping for now (MJW Dec 2015)
-            if (std::abs(subSurface.Azimuth - 360.0) < 0.01) {
-                subSurface.Azimuth = 360.0 - subSurface.Azimuth;
-            }
-            if (std::abs(baseSurface.Azimuth - 360.0) < 0.01) {
-                baseSurface.Azimuth = 360.0 - baseSurface.Azimuth;
-            }
+            // // Not sure what this does, but keeping for now (MJW Dec 2015)
```

Comment:
This code was originally added in https://github.com/NREL/EnergyPlusArchive/commit/eadb8cebe8bcee13f1f60f3e932f08c1adfcfd70. There's no mention of a specific issue it was addressing.

Reply:
I guess the original intention is trying to bring down a large angle close to 360 to something to a small angle. However, the problem is that is should equal to `baseSurface.Azimuth-360` instead (strictly speaking)---which is actually different from the angle `360-baseSurface.Azimuth`. This will introduce an error about 2 times `abs(baseSurface.Azimuth-360.0)` (which is small though)---if it has to be this way, then actually even setting the angle to zero would be better (with only 1 times `abs(baseSurface.Azimuth-360)` error introduced). 

But anyway, these lines are not needed at all, especially when the "rotational" angle difference is used (see Issue #9910 and PR #9920). I feel relatively confident that these line could (actually should) be removed without any problems.

Reply:
I am OK with this.  For our calculations, there really shouldn't be any special reason that we need to bring them to 360, and as you stated, it seems to be doing it slightly different than intended anyway.  If anyone has an issue with this, please speak up.  (@JasonGlazer?)

Reply:
I think removing this code is fine. I think it was just trying to get both angles to use the same basis if they were close to 360 degrees. So that 360.1 and 0.1 were later evaluated as the same angle. And I think the rotAzmDiffDeg() function accomplishes the same thing now. 

@jcyuan2020 since it is using abs() I don't think any error is introduced.

Reply:
The slightly problem in the original code might be the one with 359.991 degree. A 359.991 degree will be brought to a value of 0.009 (slightly smaller than 0.01). Here the error is introduced to be 2*0.009 in that that these two angles were NOT exactly the same (actually they were symmetrical with the North (0 degree) axis)---thinking about 359 degree and 1 degree for a magnified version and there would be 2 degree of an error introduced.

This is just quite a minor issue given that 0.01 (*2) would be quite small and it is usually not happening. But anyway, removing this block actually would be the clean way to eliminate the possible occurrence of a small error (especially after adapting calculating the diffs using rotational difference).